GuiFunctions/MetaDraw/PlotModelStat.cs

```diff
@@ -143,14 +146,16 @@ private void histogramPlot(int plotType)
                         dictsBySourceFile.Add(key, results.ToDictionary(p => p.Key.ToString(), v => v.Count()));
                     }
                     break;
-                case 2: // Histogram of Fragment PPM Errors TODO not shown as an option in gui
-                    xAxisTitle = "Fragment error (ppm)";
-                    binSize = 0.1;
-                    foreach (string key in psmsBySourceFile.Keys)
-                    {
-                        numbersBySourceFile.Add(key, psmsBySourceFile[key].SelectMany(p => p.MatchedIons.Select(v => v.MassErrorPpm)));
-                    }
-                    break;
+                //case 2: // Histogram of Fragment PPM Errors TODO not shown as an option in gui
```

Comment:
If this is ready to be implemented, just not displayed as an option, would you be able to add it to MetaDraw?

Reply:
Added these two plots as an option. It is now possible to parse fragment ion error from psmtsv